lib/owdit.js

```diff
@@ -96,7 +96,7 @@ function check(cwd, callback) {
   });
 
   if (result.status !== 0) {  //eslint-disable-line no-magic-numbers
-    return callback(new Error(`Error running npm ls: ${result.stderr}`));
+	console.error(`Error running npm ls: ${result.stderr}`);
```

Comment:
To elaborate further, I have a project which depends on angular-bootstrap-daterange-picker^1.0.2 which in turn has a peer dependency on jquery@2. The underlying bootstrap library behind angular-bootstrap-daterange-picker updated to support jquery@3 but the peer has not been updated yet. Everything does work in this case with jquery@3 so I am ignoring the peer dep warning when I install and it was causing owdit to exit